client/my-sites/themes/style.scss

```diff
@@ -70,10 +70,12 @@
 	0 2px 4px lighten( $gray, 20% );
 }
 
-.pulsing-dot {
-	right: 24px;
-	top: 22px;
-	left: initial;
+.web-preview {
+	.pulsing-dot {
```

Comment:
Seems like this should have some theme-specific suffix? Unless it is not specific to themes, in which case it should be in a different file.